scripts/index.js

```diff
@@ -245,56 +252,114 @@ function exportToWebsite(options) {
     })
 }
 
+function parseRequest(req/*: http.IncomingMessage*/) {
+    const bb = busboy({ headers: req.headers });
+    return new Promise/*<{ files: Array<{ rawData: Buffer; } & busboy.FileInfo>; fields: { [key: string]: string | number; }; }>*/(resolve => {
+        const files = []; //as Array<{ rawData: Buffer; } & busboy.FileInfo>;
+        const fields = {};
+        bb.on('file', (name, file, info) => {
+            const raw = [];
+            file.on('data', (data) => {
+                raw.push(data);
+            }).on('close', () => {
+                files.push({ rawData: Buffer.concat(raw), ...info });
+            });
+        });
+        bb.on('field', (name, value, info) => {
+            fields[name] = value;
+        });
+        bb.on('close', () => {
+            resolve({ files, fields });
+        });
+        req.pipe(bb);
+    });
+}
+
+function startGoldensServer() {
```

Comment:
this is great!
it is proxied by testem and once a test fails it updates the golden file